easybuild/easyconfigs/g/GROMACS/GROMACS-2018-foss-2018a.eb

```diff
@@ -0,0 +1,26 @@
+name = 'GROMACS'
+version = '2018'
+
+homepage = 'http://www.gromacs.org'
+description = """
+GROMACS is a versatile package to perform molecular dynamics,
+ i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles.
+This is CPU only build, containing both MPI and threadMPI builds.
+"""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'openmp': True, 'usempi': True}
+
+source_urls = ['ftp://ftp.gromacs.org/pub/gromacs/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['deb5d0b749a52a0c6083367b5f50a99e08003208d81954fb49e7009e1b1fd0e9']
+
+dependencies = [
+    ('hwloc', '1.11.8'),
```

Comment:
@reedts `hwloc` is already included in `foss`, there's no point in adding it as a dependency?

Reply:
@boegel I thought to have in mind that there was some trouble with GROMACS hard depending on `hwloc`. The other GROMACS easyconfigs already have the `hwloc` dependencies, so I included it too. I can of course take it out...

Reply:
@reedts The GROMACS easyconfig built with `intel` indeed include `hwloc` as dependency, because `hwloc` is not included in the `intel` toolchain (it is in `foss`, via `OpenMPI`)

Reply:
@boegel Okay, I'm taking it out then.